recipe/meta.yaml

```diff
@@ -21,6 +21,7 @@ requirements:
     - python
     - cf_units
     - fiona
+    - gdal
```

Comment:
> Also, fiona will install gdal for you, do you really need it here?

It used to install `osgeo` with this recipe, but now it is not being pulled down. `gdal` needs to be added explicitly. Are you aware of any changes that may have caused this?